lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -6,143 +6,188 @@
 from django.core.urlresolvers import reverse
 from edxmako.shortcuts import marketing_link
 from django.utils.translation import ugettext as _
+from django.utils.translation import ungettext
 
 %>
 
+<section class="wrapper confirm-enrollment shopping-cart cart-view">
+  <%block name="custom_content">
+
+  % if is_course_enrollment_closed:
+    <%
+      expiry_message = ungettext(
+          "{course_names} has been removed because the enrollment period has closed.",
```

Comment:
Because the variable is pluralized, but the verb is singular, I would add a translators comment (above the line that defines variable "expiry_message") that explains what course_names is, eg
`# Translators: course_names is a comma-separated list of one or more course names`